modules/farm_fd2_school/src/entrypoints/FD2_Application/App.vue

```diff
@@ -182,21 +178,30 @@ export default {
         this.unit.attributes.name
       );
       console.log(quantity);
+      const plantID = await farmosUtil.getPlantAsset(this.pickedPlant.uuid);
```

Comment:
Think about this name a little. You really have a full plant asset here, not just its id.